src/vs/workbench/contrib/chat/browser/actions/chatActions.ts

```diff
@@ -2027,3 +2027,69 @@ registerAction2(class EditToolApproval extends Action2 {
 		}
 	}
 });
+
+// Register actions for chat welcome history context menu
+MenuRegistry.appendMenuItem(MenuId.ChatWelcomeHistoryContext, {
+	command: {
+		id: 'workbench.action.chat.toggleChatHistoryVisibility',
+		title: localize('chat.showChatHistory.label', "✓ Chat History")
```

Comment:
@eli-w-king this is not how we indicate a menu item that is set! We have a `toggled` condition for that on the menu. I will fix it.